src/Plugin.php

```diff
@@ -113,6 +113,7 @@ public static function init() {
     WooCommerceCheckout::init();
     PlusProducts::init();
     ProductsPermalinks::init();
+    ProductDefects::init();
```

Comment:
Should we wrap this inside a check so that it would only fire on the single product template/URL? I think that would save it from running `ProductDefects::display_product_checkbox` on each request.

Reply:
Good remark, in this case and the majority of them I'd handle that logic into each module initializer so we keep this as a bootstrapper point, moreover the timing to add that condition is too early here and it won't work (I tried it) but thanks to your idea I improved the condition to bail from the method as soon as is called if is not supposed to run